processing/src/main/java/org/apache/druid/segment/column/AllTrueBitmapColumnIndex.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.segment.column;
+
+import org.apache.druid.query.BitmapResultFactory;
+import org.apache.druid.query.filter.ColumnIndexSelector;
+
+public class AllTrueBitmapColumnIndex implements BitmapColumnIndex
+{
+  private final ColumnIndexSelector selector;
+
+  public AllTrueBitmapColumnIndex(ColumnIndexSelector indexSelector)
+  {
+    this.selector = indexSelector;
+  }
+
+  @Override
+  public ColumnIndexCapabilities getIndexCapabilities()
+  {
+    return SimpleColumnIndexCapabilities.getConstant();
+  }
+
+  @Override
+  public double estimateSelectivity(int totalRows)
+  {
+    return 1.;
```

Comment:
ah, i think i was just moving the existing code around, the lineage of this is from 
```
      if (bitmapIndex == null || bitmapIndex.getCardinality() == 0) {
        return doesMatchNull() ? 1. : 0.;
      }
```
if i remember, just using `1` and `0` seems probably fine too. I'll change it to something else because i also find this to be among the stranger ways of writing it